i18n/en.json

```
@@ -2258,8 +2258,11 @@
   "announcement_bar.error.license_expiring": "Enterprise license expires on {date, date, long}.",
   "announcement_bar.error.past_grace": "Enterprise license is expired and some features may be disabled. Please contact your System Administrator for details.",
   "announcement_bar.error.preview_mode": "Preview Mode: Email notifications have not been configured.",
+  "announcement_bar.error.purchase_a_license_now": "Purchase a License Now",
   "announcement_bar.error.site_url_gitlab.full": "Please configure your [site URL](https://docs.mattermost.com/administration/config-settings.html#site-url) either on the [System Console](/admin_console/environment/web_server) or, if you're using GitLab Mattermost, in gitlab.rb.",
   "announcement_bar.error.site_url.full": "Please configure your [site URL](https://docs.mattermost.com/administration/config-settings.html#site-url) on the [System Console](/admin_console/environment/web_server).",
+  "announcement_bar.error.trial_license_expiring": "**There are {days} days left on your free trial.**",
+  "announcement_bar.error.trial_license_expiring_last_day": "**This is the last day of your free trial. Purchase a license now to continue using E10 & E10 features.**",
```

```suggestion
  "announcement_bar.error.trial_license_expiring_last_day": "**This is the last day of your free trial. Purchase a license now to continue using E10 & E20 features.**",
```

Minor text correction.